indico/modules/events/timetable/models/breaks.py

```diff
@@ -60,6 +60,10 @@ def can_access(self, user):
     def event(self):
         return self.timetable_entry.event if self.timetable_entry else None
 
+    @property
+    def session_block_obj(self):
+        return self.timetable_entry.session_block if self.timetable_entry else None
```

Comment:
this always returns None, because it's the parent entry that has a `session_block`, not the break entry itself